src/loaders/lottie/tvgLottieParser.h

```diff
@@ -35,6 +35,7 @@ struct LottieParser : LookaheadParserHandler
     }
 
     bool parse();
+    bool parseSlot(LottieComposition* comp, LottieSlot* slot);
```

Comment:
We can remove the 'comp' parameter since this parser already has a designated 'comp' (see line 40).
``` parseSlot() -> parse(LottieSlot* slot);  ```
However, I expect the ```parse(const char* slot)```,  parsing json logic sholud be in the parser. We can eliminate the dependency on RapidJSON from the overrider if the parsing tasks can be transferred to LottieParser.